CLI invocation with no args prints `-help` instead of `help` as the argument to use

```
➜  kindling git:(jdeploy) ✗ jdeploy  
Launching shellmarks gui.  Use jdeploy -help for help
Includes: [CopyRule{dir=build/libs, includes=[kindling.jar], excludes=null}]
➜  kindling git:(jdeploy) ✗ jdeploy -help
Exception in thread "main" java.lang.RuntimeException: org.apache.commons.cli.UnrecognizedOptionException: Unrecognized option: -help
        at ca.weblite.jdeploy.JDeploy.main(JDeploy.java:1939)
Caused by: org.apache.commons.cli.UnrecognizedOptionException: Unrecognized option: -help
```

Looks like `jdeploy help` does work. This is with `jdeploy@3.1.8`

--- a/cli/src/main/java/ca/weblite/jdeploy/JDeploy.java
+++ b/cli/src/main/java/ca/weblite/jdeploy/JDeploy.java
@@ -1857,7 +1857,7 @@ public class JDeploy {
             CommandLine line = parser.parse(opts, args);
             args = line.getArgs();
             if (args.length == 0 || "gui".equals(args[0])) {
-                System.out.println("Launching shellmarks gui.  Use jdeploy -help for help");
+                System.out.println("Launching shellmarks gui.  Use jdeploy help for help");
                 File packageJSON = new File("package.json");
                 if (packageJSON.exists()) {
                     JSONObject packageJSONObject = new JSONObject(FileUtils.readFileToString(packageJSON, "UTF-8"));

